controllers/edit_me/edit_me.py

```diff
@@ -0,0 +1,23 @@
+"""Simple robot controller."""
+
+from controller import Robot
+import sys
+
+# Define the target motor position in radians.
+target = 0
```

Comment:
That would be slightly better to show that the robot can move (even though that value is not enough to reach the target):
```suggestion
target = 1
```